### ISSUE TITLE
vae.add_loss(vae_loss) on line# 57 gives an attribute error : 'Tensor' object has no attribute 'add_loss'

### ISSUE THREAD

**Issue by dsadulla** [llSourcell/autoencoder_explained#2]
Hi,

I tried to reproduce your results but I end up with an error

`vae.add_loss(vae_loss) `

on line# 57 gives an attribute error :

` 'Tensor' object has no attribute 'add_loss'`

Found the add_loss definition within keras.engine.topology but even after importing add_loss explicitly, I was not able to solve this.

Is this a version issue of Keras and or Tensorflow? FYI I am using Keras == 2.1.1 and TensorFlow == 1.4.0 in Jupyter notebook environment. 

Let me know if you need more info.

Thanks!
Deepak

**Comment by dsadulla** [comment]
The error was resolved, because I had misspelled a few arguments

**Comment by anuragdeshmukh** [comment]
hello,
can you please clarify how you corrected the issue? I'm getting a similar issue.
'Model' object has no attribute 'add_loss'

**Comment by dsadulla** [comment]
Check this issue
https://github.com/llSourcell/autoencoder_explained/issues/1#issuecomment-361073825

It redirects to the original source ie fchollet's keras examples.

Btw, a simpler approach would be to remove line add_loss and use argument loss='vae_loss' in the compile step.

Hope this helps..
